cinema_service/settings.py

```diff
@@ -20,9 +20,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
```

Comment:
The SECRET_KEY is currently hardcoded and exposed in the settings file. This should be kept secret and ideally loaded from an environment variable to enhance security, especially in a production environment.